fem/bilininteg.hpp

```diff
@@ -3071,6 +3071,22 @@ class VectorScalarProductInterpolator : public DiscreteInterpolator
    VectorCoefficient *VQ;
 };
 
+/** Interpolator of the 2D cross product between a vector coefficient and an
+    H(curl)-conforming field onto an L2-conforming field. */
+class ScalarCrossProductInterpolator : public DiscreteInterpolator
```

Comment:
Add to the documentation in https://mfem.org/lininterp/?

Reply:
Will do, thanks!

Reply:
Just created PR [#88](https://github.com/mfem/web/pull/88) to add documentation for this (and the enhanced curl interpolator).

Reply:
Merged, thanks!